doc/refman/Universes.tex

```diff
@@ -131,6 +131,52 @@
   polymorphically, not at a single instance.
 \end{itemize}
 
+\asection{{\tt Cumulative, NonCumulative}}
+\comindex{Cumulative}
+\comindex{NonCumulative}
+\optindex{Inductive Cumulativity}
+
+Inductive types, coinductive types, variants and records can be
+declared cumulative using the \texttt{Cumulative}. Alternatively,
+there is an option \texttt{Set Inductive Cumulativity} which when set,
+makes all subsequent inductive definitions cumulative. Consider the examples below.
+\begin{coq_example*}
+Polymorphic Cumulative Inductive list {A : Type} :=
```

Comment:
The `Cumulative` prefix alone doesn't work. One must also use `Polymorphic`. I think this is a bug and `Cumulative` should always imply `Polymorphic` (given that it makes no sense otherwise). At the very least, using `Cumulative` in a monomorphic setting should produce a warning.